GoogleForCreators/web-stories-wp
# Safari page previews: some shapes display incorrectly

## Context



This PR fixes an issue where some shapes appear incorrectly on the pagepreview in Safari.

## Summary



The culprit here is buggy interpretation of `overflow` by Safari.  Having `overflow: hidden;` on the parent causes the shapes  to be truncated.  

## Relevant Technical Choices



## To-do



## User-facing changes



| Before | After |
|--------|-------|
|   ![Screenshot 2021-12-07 at 15 31 52](https://user-images.githubusercontent.com/276316/145062254-512cd231-d4c1-4ffa-988f-d2e85f81f39a.png)     |   ![Screenshot 2021-12-07 at 16 58 16](https://user-images.githubusercontent.com/276316/145062322-8b004b70-e829-4a40-b655-29f319d04869.png)    |

## Testing Instructions






This PR can be tested by following these steps:

1. Create/Open a story in Safari
2. Add some different shapes to a page e.g. Heart & Pentagon shapes
3. Check the page preview that the shape are shown correctly


## Reviews

### Does this PR have a security-related impact?


No

### Does this PR change what data or activity we track or use?


No

### Does this PR have a legal-related impact?


No

## Checklist




---



Fixes #9173
